setup.py

```diff
@@ -7,7 +7,7 @@
     name="pytest-flyte",
     version="0.0.0+dev0",
     packages=find_packages("src"),
-    entry_points={"pytest11": ["flyte = pytest_flyte.plugin"]},
+    entry_points={"pytest11": ["flyte = pytest_flyte"]},
```

Comment:
this just reverts a change I made earlier with another issue that `pytest-docker` is loaded via pytest plugin load order after `pytest-flyte`, which means that none of the pytest flyte fixtures were being used in my tests...

fixed this with `pytest.ini`
```
[pytest]
addopts = -p docker -p flyte

```

Reply:
Yea these are the kinds of things we should avoid having the users do. Perhaps its worth implementing a small subset of the pytest-docker's functionality in this plugin.